src/main/java/be/seeseemelk/mockbukkit/entity/PlayerMockFactory.java

```diff
@@ -1,67 +1,45 @@
 package be.seeseemelk.mockbukkit.entity;
 
-import java.util.HashSet;
-import java.util.Random;
-import java.util.Set;
-import java.util.UUID;
-
 import be.seeseemelk.mockbukkit.ServerMock;
+import org.jetbrains.annotations.NotNull;
 
-public final class PlayerMockFactory
-{
-	private static final String[] FIRST_NAMES = { "James", "Mary", "John", "Particia", "Robert", "Jennifer", "Michael", "Elizabeth", "William", "Linda" };
-	private static final String[] LAST_NAMES = { "Smith", "Johnson", "Jones", "Brown", "Davis", "Miller", "Wilson", "Moore", "Taylor", "Anderson" };
+import java.util.Random;
```

Comment:
```suggestion
```
Import not used

Reply:
But this import is used at line 11

Reply:
It is still used by the Random property at line 11, which is itself still used.